### ISSUE TITLE
Can't connect to the database

### ISSUE THREAD

**Issue by sgoings** [deis/workflow#455]
Workflow is failing to start because it can't connect to the database:

```
kubectl logs -p --namespace=deis deis-workflow-fcs7h
system information:
Django Version: 1.9.3
Python 3.5.1
Django checks:
System check identified no issues (2 silenced).
Health Checks:
Checking if database is alive
There was a problem connecting to the database
```

@helgi investigated a bit further than I did and found this further information:
```
            return self.sign(operation_name, request)
          File "/usr/local/lib/python2.7/dist-packages/botocore/signers.py", line 123, in sign
            signature_version)
          File "/usr/local/lib/python2.7/dist-packages/botocore/signers.py", line 163, in get_auth_instance
```

**Comment by helgi** [comment]
That's not a workflow problem, that log is from the database

**Comment by sgoings** [comment]
Ok - so the real problem is that workflow can't connect to the database because the database is having problems with botocore oriented logic?

**Comment by helgi** [comment]
Yeah

**Comment by sgoings** [comment]
Thanks @helgi. Will be addressed by deis/postgres#47
